dts/bindings/memory-controllers/atmel,sam-smc.yaml

```diff
@@ -148,3 +148,17 @@ child-binding:
         is encoded in 9 bits where the two highest bits are multiplied
         with an offset of 256.
         Effective value for each element: 256 x cycle[8:7] + cycle[6:0]
+
+    atmel,smc-bus-width:
+      type: int
+      default: 8
+      description: |
+        Bus Width configuration for the Static Memory Controller that
+        is wired to the external device.
+        SMC on SAMx7 supports 8/16-bit bus width, on SAM4
+        only 8-bit data buses are supported.
```

Comment:
We need to change the below comment because the SAM3X supports 8/16-bit bus width too.

Suggestion:

```diff
-        is wired to the external device.
-        SMC on SAMx7 supports 8/16-bit bus width, on SAM4
-        only 8-bit data buses are supported.
+        is wired to the external device. The SMC supports 8/16-bit bus
+        width with exception of SAM4 devices which only supports 8-bit.
```